fluss-flink/fluss-flink-1.18/src/main/java/org/apache/fluss/flink/adapter/MultipleParameterToolAdapter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.fluss.flink.adapter;
+
+import org.apache.flink.api.java.utils.MultipleParameterTool;
+
+import java.util.Map;
+
+/** A adapter for flink 1.x {@link MultipleParameterTool} . */
+public class MultipleParameterToolAdapter {
```

Comment:
We don't need to add the adapter for the 1.18, 1.19, 1.20 modules, because they are the same code with the `fluss-flink-common`, and no need to override the class. We only need to add the adapter for 2.1.